vmngclient/api/templates.py

```diff
@@ -161,7 +168,7 @@ def device_to_cli(self, device: Device) -> bool:
             "devices": [{"deviceId": device.uuid, "deviceIP": device.id}],
         }
         endpoint = "/dataservice/template/config/device/mode/cli"
-        response = self.session.post(url=endpoint, data=payload).json()
+        response = cast(dict, self.session.post(url=endpoint, json=payload).json())
```

Comment:
Do we need to cast response on dict? `json()` already converts it into Python dict.